### ISSUE TITLE
SectionList

### ISSUE THREAD

**Issue by RWOverdijk** [akveo/react-native-ui-kitten#952]
## 🚀 Feature Proposal

Having a sectionlist of sorts would be a great addition.

https://reactnative.dev/docs/sectionlist

PS: are there any UI Kitten plugins out there? UI Kitten would be perfect for such a thing.

**Comment by artyorsh** [comment]
What do you mean with plugins? :)

**Comment by RWOverdijk** [comment]
@artyorsh Components built on the eva system that fit in the ui-kitten style. You know stuff like a sectionlist, card stack, slide menu... Things 😄

**Comment by artyorsh** [comment]
Good idea, and we discussed it internally. However, we’re not ready to maintain another ui kit based on this one.
There is a [demo app](https://github.com/akveo/kittenTricks)

**Comment by RWOverdijk** [comment]
@artyorsh Oh no, I didn't mean another kit. I meant a list of components contributed by others. Maybe a markdown file or something.

**Comment by lesmo** [comment]
I've used `SectionList` with `ui-kitten`, and even in `FlatList` without using the `List` component. It [doesn't seem to be doing much under the hood](https://github.com/akveo/react-native-ui-kitten/blob/master/src/components/ui/list/list.component.tsx), so I've never faced issues using `ListItem` on it's own.

I think it would be a good idea to add some sort of wrapper for `SectionList`, maybe even have some default component styled by `ui-kitten` to be used as section.

> @artyorsh Oh no, I didn't mean another kit. I meant a list of components contributed by others. Maybe a markdown file or something.

Never stumbled upon something like that so far. It would be nice, though. We've built a couple of things on top of `ui-kitten` but I don't think they're generic enough (or they're just a simple tweak) to contribute as "plug-ins".
